### PR TITLE
bump doctrine\dbal to ^3.1

### PR DESCRIPTION
**Pull request by eddeh**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "php": ">=8.0",
         "doctrine/common": "^3.1",
         "doctrine/persistence": "^2.2",
-        "doctrine/dbal": "^2.13",
+        "doctrine/dbal": "^3.1",
```
Comment (Kocal) [inline composer.json:17]: Would it be possible to use `^2.13 || ^3.1`?

Comment (eddeh) [reply composer.json:17]: Sure.
I have used single | to be consistent.

**Comment by mmarton** [comment]
Can we move this PR forward? I would like to upgrade to dbal 3 and this is my last dependency that's locked to dbal 2.

**Comment by TomasVotruba** [comment]
Of course. The CI must pass before merging

**Comment by mmarton** [comment]
I'm not familiar with rector but looks like a simple rebase would solve the issue.

@eddeh could you update the PR please?

**Comment by eddeh** [comment]
@mmarton @TomasVotruba 
Sorry about this. I wasn't aware I had to check rector too, it's my bad I did not read Contributing part properly.

I believe it's good to go.

**Comment by TomasVotruba** [comment]
The green the better :) 

Thank you :+1:

**Comment by simivar** [comment]
Is there possibility to release a new version with that change? :)

**Comment by TomasVotruba** [comment]
Sure, gimme a sec :)

**Comment by eddeh** [comment]
GG @TomasVotruba

**Comment by TomasVotruba** [comment]
There is it https://github.com/KnpLabs/DoctrineBehaviors/releases/tag/2.5.0

**Pull request merged**